Library/brew.sh

```diff
@@ -70,6 +70,10 @@ then
     HOMEBREW_RUBY_PATH="/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby"
   else
     HOMEBREW_RUBY_PATH="$(which ruby)"
+    if [[ -z "$HOMEBREW_RUBY_PATH" ]]
```

Comment:
If you wanted to improve the error message to be more helpful you could suggest some commands to install Ruby or point them to http://linuxbrew.sh#dependencies. I suggest the latter. Eventually I'm hoping to offer to install Ruby for the user at this point using one of the host's package manager, a bottle (which will be a bit tricky without ruby, but possible), or possibly [Traveling Ruby](http://phusion.github.io/traveling-ruby/).